src/ducks/billsMatching/testUtils.spec.js

```diff
@@ -0,0 +1,71 @@
+const { mkLineParser, parseTable } = require('./testUtils')
```

Comment:
Not necessary to review the whole file in this commit, it is basically a copy/paste from cozy-konnector-libs at this stage.